pkgs/build-support/fetchgit/default.nix

```diff
@@ -97,5 +112,10 @@ stdenvNoCC.mkDerivation {
   passthru = {
     gitRepoUrl = url;
   };
-}
+};
+in
+if verifyCommit || verifyTag then
+  (import ./verify.nix { inherit lib runCommand writeShellApplication writeText git openssh gnupg; } { inherit name rev verifyCommit verifyTag publicKeys leaveDotGit fetchresult; })
```

Comment:
I thought about using `callPackage` here but I can't say which would be idiomatic.